### PR TITLE
SAIC-448 Added test 'image which belongs to deleted tenant migration'

### PR DESCRIPTION
**Pull request by ntymtsiv**

### CODE REVIEW

File: devlab/tests/config.py [hunk 1]
```diff
@@ -1,3 +1,5 @@
+img_url = 'http://download.cirros-cloud.net/0.3.3/cirros-0.3.3-x86_64-disk.img'
```
Comment (realsystem) [inline devlab/tests/config.py]: Do you need a real image for testing migration? As I know you can touch an empty file and use it for glance. Or just use raw data in API. Please, see https://github.com/MirantisWorkloadMobility/CloudFerry/pull/383/files for reference.

**Comment by mirrorcoder** [comment]
Looks good. +1

File: devlab/tests/config.py (hunk 1 above)
Comment (ntymtsiv) [inline devlab/tests/config.py]: We use created images to spawn vms. For testing migration we need more than 1 image.

**Comment by roman-verchikov** [comment]
looks ok, can be merged
